lib/experimental/connection-sources/index.php

```diff
@@ -12,4 +12,7 @@
 		// if it doesn't, `get_post_meta()` will just return an empty string.
 		return get_post_meta( $block_instance->context['postId'], $meta_field, true );
 	},
+	'pattern_attributes' => function ( $block_instance, $meta_field ) {
```

Comment:
It would be great to get confirmation from @michalczaplinski or @SantosGuillamot on how this array should be shaped. There is `name` key defined with the `meta` value, which implies, we should have another entry with the name `pattern`. I might be completely wrong here.

Reply:
I think that nothing was ultimately decided here! Just for reference, this is a previous discussion with Riad: https://github.com/WordPress/gutenberg/pull/53247#discussion_r1293580463

We'll need a nested array here OR have a one file per source (like in Riad's [original prototype](https://github.com/WordPress/gutenberg/pull/51375/files#diff-04639b3e641a433561e5a70976aeb456c0e5f8199d2d2e05227fd9302b376a5e)) so `meta.php`, `patterns.php`, etc.